AppIntro.js

```diff
@@ -455,4 +478,9 @@ AppIntro.defaultProps = {
   doneBtnLabel: 'Done',
   skipBtnLabel: 'Skip',
   nextBtnLabel: '›',
+  defaultIndex: 0,
+  showSkipButton: true,
+  showDoneButton: true,
+  paginationStyles: {},
+  dotStyles: {}
```

Comment:
I need custom pagination/dot styles, but I agree on the custom styles. I think that will work better